star_tides/api/controllers/__init__.py

```diff
@@ -0,0 +1,35 @@
+''' star_tides.api.controllers
+'''
+import json
+from typing import NamedTuple
+from flask import Response
+from star_tides.exceptions import StarTidesException
+
+
+def build_response(response: NamedTuple) -> Response:
```

Comment:
If you're going to check the `response.response` field, maybe subclass the NamedTuple param? You could do something like the `Response.for_error` and `Response.for_response` or something so that only one of error/response is set.

Reply:
I'm not sure what you mean, but I don't think it's an issue any more because I deleted that code.